nextflow.config

```
@@ -31,7 +35,8 @@ params {
   af_resolution = 'cr-like-em' // alevin-fry quant resolution method: default is cr-like, can also use full, cr-like-em, parsimony, and trivial
   repeat_mapping = false // if alevin or salmon mapping has already been performed and output files exist, mapping is skipped by default. Use `--repeat_mapping` to perform mapping again
   repeat_gdemux = false // if genetic demultiplexing has been performed and output files exist, genetic demux is skipped by default. Use `--repeat_gdemux` to run these steps. 
-  
+  publish_fry_outs = false // alevin-fry outputs are not published by default. Use `--write_fry_outs` to publish these files to the `checkpoints` folder. 
```

Making the comment match.
```suggestion
  publish_fry_outs = false // alevin-fry outputs are not published by default. Use `--publish_fry_outs` to publish these files to the `checkpoints` folder. 
```